CONTRIBUTING.md

```diff
@@ -2,39 +2,47 @@
 
 ## Coordination, communication and community
 
-Many ember-paper contributor hang out on the [ember-paper channel on slack](https://embercommunity.slack.com/messages/ember-paper/). Not a slack user? [Get your invitation.](https://ember-community-slackin.herokuapp.com/)
+Many ember-paper contributors hang out on the [ember-paper channel on slack](https://embercommunity.slack.com/messages/ember-paper/). Not a slack user? [Get your invitation.](https://ember-community-slackin.herokuapp.com/)
 
 ## Coding style
 
-* ember-paper uses the [ember-suave](https://github.com/DockYard/ember-suave) coding style.
+* **Suave.** ember-paper uses the [ember-suave](https://github.com/DockYard/ember-suave) coding style.
 
-* Before submitting a pull request,
+* **jscs.** Before submitting a pull request,
 check for coding style issues with  `jscs -c .jscsrc app addon`.
 
-* Require action closures rather than strings representing action names.
+* **Actions.** Require action closures rather than strings representing action names.
 `{{some-component someAction=(action "myAction")}}`, not `{{some-component someAction="myAction"}}`.
 
-## Capitalization and naming
+* **Encapsulation.** When communicating with a private ember-paper component, bind as many properties as are needed.
+When communicating between two public ember components, use `nearestOfType` to find the outer component,
+so that the user does not have to bind many properties that are implementation details.
+For example, `ember-paper-inner` is a private component:
+```hbs
+{{ember-paper-inner parent=parent fullscreen=fullscreen onOutsideClick=onOutsideClick}}
```

Comment:
I understand the example in question, but I think that for a newcomer, `parent` may be confusing. Someone might think it is a reference to the parent component, which is exactly what we're trying to avoid.